go-controller/hybrid-overlay/pkg/controller/node_linux.go

```diff
@@ -269,6 +276,20 @@ func (n *NodeController) AddNode(node *kapi.Node) error {
 		klog.Infof("Add hybridOverlay Node %s", node.Name)
 		err = n.hybridOverlayNodeUpdate(node)
 	}
+	if atomic.LoadUint32(&n.initialized) == 1 {
+		pods, err := n.podLister.List(labels.Everything())
```

Comment:
shouldn't you only list pods on this node?

Reply:
we do only list pods in the node, we get passed a localPodLister. I want to make an additional PR that cleans that up and renames the podLister after in case I need to cherry pick this one.